backend/src/main/java/de/bund/digitalservice/ris/caselaw/adapter/JurisXmlExporterResponseProcessor.java

```diff
@@ -67,23 +69,26 @@ private void processInbox(Store store) {
       Folder inbox = store.getFolder("INBOX");
       inbox.open(Folder.READ_WRITE);
 
-      List<MessageWrapper> processedMessages =
-          Arrays.stream(inbox.getMessages())
-              .map(wrapperFactory::getResponsibleWrapper)
-              .flatMap(Optional::stream)
-              .sorted(
-                  Comparator.comparing(wrapper -> wrapper instanceof ImportMessageWrapper ? 0 : 1))
-              .map(this::forwardMessage)
```

Comment:
Vielleicht hier brauche ich nochmal eine Erklärung, warum wird diese Methode readEmails so oft aufgerufen. Ging es nur um diese eine Dokument XXRE000000224 dass nicht gefunden wurde.

Reply:
We have a scheduled job which look every 30 minutes into the inbox and get all mails and process these. Successful mail moved to a "processed"-folder. Failed mail will be tried after 30 minutes again.